storage/replica_command.go

```diff
@@ -986,7 +986,7 @@ func (r *Replica) splitTrigger(batch engine.Engine, split *proto.SplitTrigger) e
 	// Add the new split range to the store. This step atomically
 	// updates the EndKey of the updated range and also adds the
 	// new range to the store's range map.
```

Comment:
Sorry, I forgot I already merged the PR. Will send a small PR to update the comment.